.github/workflows/pr-build-test-examples.ps1

```diff
@@ -2,7 +2,7 @@ $ErrorActionPreference = "Stop";
 $PSNativeCommandUseErrorActionPreference = $true;
 
 try {
-    ./gradlew allTests `
```

Comment:
> Made CI test scripts use ./gradlew allTests instead of ./gradlew clean.

What does this address?

Reply:
That's to test CargoCleanTask, which needs *some magic* to implement locks.

https://gitlab.com/trixnity/uniffi-kotlin-multiplatform-bindings/-/merge_requests/52